nested/appgw.json

```diff
@@ -26,7 +26,7 @@
             "type": "Microsoft.Network/applicationGateways",
             "name": "[parameters('lampCommon').appGwName]",
             "location": "[parameters('lampCommon').location]",
-            "apiVersion": "2018-02-01",
+            "apiVersion": "latest",
```

Comment:
Let's not use "latest" for apiVersion, let's have a unique date string around this commit/time-frame for every applicable mention of apiVersion.